src/components/webviews/jsInteractions/jsLogInterception.spec.tsx

```diff
@@ -1,16 +1,26 @@
-/* eslint-disable no-console */
+/* eslint-disable no-console */ // we're mocking console
 import { tryConsole } from './jsLogInterception'
 
 const logId = 'logId'
 const logger = {
   log: jest.fn(),
   debug: jest.fn()
-}
-const makePayload = ({ type, args }) => ({
+} as unknown as MiniLogger
+const makePayload = ({
+  type,
+  args
+}: {
+  type: string
+  args?: string[]
+}): {
+  nativeEvent: {
+    data: string
+  }
+} => ({
```

Comment:
Those types should definitively be extracted somewhere else. Inlining them make the code very difficult to read.

Reply:
updated